docs/source/en/tasks/video_classification.md

```diff
@@ -104,6 +104,31 @@ UCF101_subset/
         ...
 ```
 
+Let's now count the number of total videos we have.
```

Comment:
The case of documentation is slightly different than notebooks demos - we'd being instructive rather than taking the user step-by-step 

```suggestion
You can then count the number of total videos.
```